---
title: "Initial Proposal"
author: "Team GGJJ"
date: January 23 2019
output: github_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

## Title of project

This is initial proposal for **Team Brocoli**.

Our team members are: 

Github ID |  Name
---------|------------
[@garyzhubc](https://github.com/garyzhubc) | Gary / Peiyuan Zhu (MSc, Statistics)
[@glenn-mcguinness](https://github.com/glenn-mcguinness) | Glenn McGuinness (MSc, Statistics)
[@janxue](https://github.com/janxue) | Jan Xue (MSc, Botany)
[@j-schaub](https://github.com/j-schaub) | Jessica Schaub (MSc, Oceanography)
[@ymkng](https://github.com/ymkng) | Michelle Kang (MSc, Bioinformatics)


Our project is based on the 2018 paper [Transcriptome and DNA methylome reveal insights into yield heterosis in the curds of broccoli (Brassica oleracea L var. italic)](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-018-1384-4?fbclid=IwAR3glcIlScIMQzxMjWtuaDowqrv7DlZbzAjrTAiHRlXkRfWdyfWgt_BqCW4).

*Refference*

BMC Plant Biology **18**:168
https://doi.org/10.1186/s12870-018-1384-4

## Proposal

The following proposal is meant to give an outline of our project with an introduction, objectives, description of the data, proposed major analysis, and a breakdown of assigned tasks.

### Abstract

Hybrid vigour or heterosis is the improved quality or trait in the offspring as a result of mixing the genetic contributions of its parents. As an illustration, brocoli have flowers that pack densely to form the head which is the main edible part of the plant. When breeding disimillar strains of brocoli their offspring can form much larger heads of brocoli as demonstrated by figure 1. 

![Brocoli heterosis](https://scontent.fyka1-1.fna.fbcdn.net/v/t1.0-9/49213408_10218077757916241_2440798217458155520_n.jpg?_nc_cat=111&_nc_ht=scontent.fyka1-1.fna&oh=d67a2a1ec54157595a7320d8cbed3bc3&oe=5CFB074F).

Figure 1. Hybrid vigour demonstrated by the larger head size of the hybrid offspring in comparison to the parents.

In order to explore the genetic and methylomic basis of this phenomenon as well as the contribution from each parent we will be analyzing RNA-seq and Whole Genome Bisulfite Sequencing (WGBS) data from [Li et al. 2018](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-018-1384-4) from parent and offspring brocoli that exhbited hybrid vigour. 
 
The data transcriptomic and methylomic data associated with this paper can be found [here](https://static-content.springer.com/esm/art%3A10.1186%2Fs12870-018-1384-4/MediaObjects/12870_2018_1384_MOESM2_ESM.zip).

<We will first explore the data through graphical visulaization, for example heat maps of RNA seq data. After that to compare the different gene expressions and epigenetic characteristics we will use ANOVA or analysis of variance for parental and hybrid offspring data.> --> Gary and Glenn??? maybe add more


### Motivation 

[Li et al. 2018](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-018-1384-4) demonstrate that:


Male rats have lower DNMT3a activity and DNA methylation than females
Inhibiting DNMT masculinized neuronal markers and sexual behaviour in female rats, and has no effect on male behaviour. Females with conditional knockout of isoform DNMT3 also display male sexual behaviour
Even outside of the restricted period of development where they are sensitive to hormone therapy, DNMT inhibition still masculinizes females
Some changes in gene expression are as a result of DNMT inhibition.
The authors did not, however, attempt to link these changes to their methylation data which is why we have selected to further evaluate this paper.


- a
- b
- methylation site differences were greater between parent-parent than parent-hybrid.

### Goals

based on the data collected from --- and --- study, we will :

1. Find genes that were differentially expressed between parents and offspring. Instead of treating them sepparately, we will try pooling the parents and pooling the offspring data. < unsure about this>

2. Do the differentially expressed genes in the hybrid offspring also correlate with close by methylation differences. If so how which genes?

3.

### Description of Data

the data provided from x paper :

- RNA seq data with x. the samples included are :
- a
- b
- c


### Methods and Division of Labour

- Test whether the parental lines more simmilar to each other than to the offspring with regards to the expression data? how about the methylation data? (cluster analysis)
- Find differentially expressed genes with two approaches. First try between each parent child triad. Then try again by pooling the parental data and pooling the child data. How different are these results?
- 